lib/sql.js

```diff
@@ -619,18 +667,38 @@ SQLConnector.prototype.buildUpdate = function(model, where, data, options) {
  * @param {Function} cb The callback function
  */
 SQLConnector.prototype.update = function(model, where, data, options, cb) {
+  var stmt = this._buildUpdate(model, where, data, options);
+  this._executeAlteringQuery(model, stmt.sql, stmt.params, options, function(err, data) {
+    if (cb)
+      cb(err, data);
```

Comment:
Same comment as above, the second arg should be `info` instead of `data`, but it's even better to not create a new anonymous function at all:

```js
this._executeAlteringQuery(model, stmt.sql, stmt.params, options, cb || NOOP);
```